CHANGELOG.md

```diff
@@ -2250,4 +2251,4 @@
 
 # cuDF 0.2.0 and cuDF 0.1.0
 
-These were initial releases of cuDF based on previously separate pyGDF and libGDF libraries.
\ No newline at end of file
+These were initial releases of cuDF based on previously separate pyGDF and libGDF libraries.
```

Comment:
Unintended change?

Reply:
Looks like this is from VSCode's `"files.insertFinalNewline": true` setting...